sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -83,16 +205,37 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] {
       } else {
         withFilter
       }
-
       withProjection
   }
+
+  private def getWrappedScan(
+      scan: Scan,
+      sHolder: ScanBuilderHolder,
+      aggregation: Option[Aggregation]): Scan = {
+    scan match {
+      case v1: V1Scan =>
+        val pushedFilters = sHolder.builder match {
+          case f: SupportsPushDownFilters =>
+            f.pushedFilters()
+          case _ => Array.empty[sources.Filter]
+        }
+        V1ScanWrapper(v1, Array.empty[sources.Filter], pushedFilters, aggregation)
```

Comment:
`V1ScanWrapper.translatedFilters` is always Nil now?

Reply:
It seems like only for display purposes. I'm OK to remove it but let's do it more explicitly and remove this field from `V1ScanWrapper`